dotnet/samples/GettingStarted/RepoUtils/TextOutputHelperExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Xunit.Abstractions;
+
+namespace Examples;
+
+public static class TextOutputHelperExtensions
+{
+    public static void WriteLine(this ITestOutputHelper testOutputHelper, object target)
+    {
+        testOutputHelper.WriteLine(target.ToString());
+    }
+
+    public static void WriteLine(this ITestOutputHelper testOutputHelper)
+    {
+        testOutputHelper.WriteLine(string.Empty);
+    }
+
+    public static void Write(this ITestOutputHelper testOutputHelper)
+    {
+        testOutputHelper.WriteLine(string.Empty);
```

Comment:
Shouldn't this be Write?

Reply:
@westey-m , `ITestOutputHelper` does not support `Write`, only `WriteLine`, this is just to make easier when the developer wants to convert to a `Console` application.